package/contents/ui/configGeneral.qml

```diff
@@ -1,16 +1,25 @@
-import QtQuick 2.0
+import QtQuick 2.15
```

Comment:
is this really needed?

Reply:
This is needed to use RegularExpressionValidator